app/actions/index.js

```diff
@@ -276,3 +280,46 @@ export const listRemoteSurveys = () => (dispatch, getState) => {
       );
   });
 };
+
+export const initializeObservation = tags => dispatch =>
+  dispatch({
+    type: types.INITIALIZE_OBSERVATION,
+    tags
+  });
+
+export const updateObservation = tags => dispatch =>
+  dispatch({
+    type: types.UPDATE_OBSERVATION,
+    tags
+  });
+
+export const saveObservation = () => (dispatch, getState) => {
+  const { observation } = selectActiveObservation(getState());
+
+  dispatch({
+    type: types.SAVING_OBSERVATION,
+    observation
+  });
+
+  // TODO this is wrong
+  // observation currently looks like (still needs a point / associated feature):
+  // {
+  //   tags: {
+  //     highway: "residential",
+  //     name: "My Street",
+  //     surface: "paved"
+  //   }
+  // }
+  return osm.createObservation(observation, error => {
```

Comment:
Weird, I'm not sure why it's written that way. It looks like the codebase doesn't use it, either. Let's just replace it with a passthrough function that takes the same params as `osm-p2p-observations`.